mqtt/mqtt-broker/config/with_tls.json

```diff
@@ -1,15 +1,11 @@
 {
-    "transports": [
-        { 
-            "tcp": {
-                "address": "0.0.0.0:1883"
-            } 
+    "transports": {
+        "tcp": {
+            "address": "0.0.0.0:1883"
         },
-        { 
-            "tls": { 
-                "address": "0.0.0.0:8883", 
-                "certificate": "identity.pfx" 
```

Comment:
Can the broker still use a custom certificate?

Reply:
yes. the idea is the broker in edgehub mode downloads cert from edgelet and puts it in the filesystem. later when transport starts it reads cert from the file and uses it to accept TLS connections